src/interfaces.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=C0115, C0116
```

Comment:
I know that more descriptive names of the rules exists 🤔 Maybe we should use one of these.